src/plugins/vis_type_vega/public/vega_view/vega_base_view.js

```diff
@@ -437,7 +437,7 @@ export class VegaBaseView {
    * Set global debug variable to simplify vega debugging in console. Show info message first time
    */
   setDebugValues(view, spec, vlspec) {
-    this._parser.searchAPI.inspectorAdapters?.vega.bindInspectValues({
+    this._parser.searchAPI.inspectorAdapters?.vega?.bindInspectValues({
```

Comment:
What are the cases when `vega` property is undefined?

Reply:
It cant get passed in from the expression function method

Reply:
Will add a note why this is needed

Reply:
Why can the inspector now be null exactly?

Reply:
Yeah, this feels like something out of scope, but that we'll want to add an issue to track - ideally vega inspect values are always bound, although I'm not too sure of the behavior in an embedded context. Maybe we only use them in the vega editor environment.

Reply:
To support this, we would need fairly hacky code. At the build pipeline, the visualization plugin checks the chart type, which it thinks is line chart and injects the inspector adapter for line chart. We would have to try trick it for when we want to render using vega that it is actually a vega chart and to inject the vega inspector. It would be best to not support this and work on a better long term solution.

Reply:
+1, we discussed offline. The only impact would be the vega debugger wouldn't be shown when inspecting a line chart  with associated events. Also, trying to add the vega inspector in visualize_embeddable would require an odd (maybe impossible) dependency of visualizations taking on vis_type_vega as a dep.

Reply:
Yeah agreed - would be better to have a general issue as part of the vega-lite migration.